nexus/lib/qmcpack.py

```diff
@@ -284,7 +284,12 @@ def process_jastrow(wf):
                     jd = dict()
                     for j in js:
                         jtype = j.type.lower().replace('-','_').replace(' ','_')
-                        jd[jtype] = j
+                        key = jtype + j.name
+                        if key in jd.keys():
+                            msg = 'duplicate jastrow in '+self.__class__.__name__
+                            self.error(msg)
+                        #end if
+                        jd[key] = j
```

Comment:
The problem is exactly that I want to use two one-body Jastrows with different cutoffs.
Please see the attached input in #2037. Basically, the first J1 sr has rcut="1" and fixes the e-p cusp, whereas the second J1 lr has rcut="L/2" to fix charge density at longer range. Without this split, the optimized variance is twice as high.

Reply:
Yes, I want the two J1 case to be supported.  Just update the code so that `key=jtype` if there is only a single J1 (as originally) and `key=jtype+j.name` if there are two or more J1's.

Reply:
@jtkrogel Sorry I misunderstood. I just pushed an update based on your comment.